src/github.com/couchbase/sync_gateway/base/bucket.go

```diff
@@ -107,23 +107,23 @@ func (bucket CouchbaseBucket) View(ddoc, name string, params map[string]interfac
 	return vres, bucket.Bucket.ViewCustom(ddoc, name, params, &vres)
 }
 
-func (bucket CouchbaseBucket) StartTapFeed(args sgbucket.TapArguments) (sgbucket.TapFeed, error) {
+func (bucket CouchbaseBucket) StartTapFeed(args sgbucket.TapArguments, notify sgbucket.BucketNotifyFn) (sgbucket.TapFeed, error) {
```

Comment:
Is there a reason to not use sgbucket.TapArguments to store/share the notify function?